src/pages/organize/[orgId]/projects/[campId]/events/[eventId]/participants.tsx

```diff
@@ -41,15 +42,23 @@ const ParticipantsPage: PageWithLayout<ParticipantsProps> = ({
   const dataModel = useModel(
     (env) => new EventDataModel(env, parseInt(orgId), parseInt(eventId))
   );
+
   return (
     <ZUIFuture future={dataModel.getData()}>
-      {() => {
+      {(data) => {
         return (
-          <Grid container spacing={2}>
-            <Grid item md={8} xs={12}>
-              <ParticipantSummaryCard model={dataModel} />
+          <>
+            <Grid container spacing={2}>
+              <Grid item md={8} xs={12}>
+                <ParticipantSummaryCard model={dataModel} />
+              </Grid>
             </Grid>
-          </Grid>
+            <EventParticipansList
+              data={data}
+              model={dataModel}
+              orgId={parseInt(orgId)}
+            ></EventParticipansList>
```

Comment:
Oh I just noticed this little mistake as well. The empty tag is not closed as I would expect. Maybe it's a good idea to add the [`self-closing-comp` linting rule](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/self-closing-comp.md)? Would you mind adding it as part of this PR?

Reply:
Great idea! I'll implement it!